src/BizHawk.Client.Common/savestates/ZipStateSaver.cs

```diff
@@ -8,9 +8,16 @@ namespace BizHawk.Client.Common
 	public class ZipStateSaver : IDisposable
 	{
 		private readonly IZipWriter _zip;
-		private bool _isDisposed;
+		private bool _isDisposed;
+
+		private static void WriteZipVersion(Stream s)
+		{
+			var sw = new StreamWriter(s);
+			sw.WriteLine("0"); // version 1.0.0
+			sw.Flush();
```

Comment:
Probably reads better with a `using var sw = new StreamWriter(s);`; which should also nullify the need for flushing.